src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,105 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private static final int SIZE_MULTIPLIER = 2;
+    private int threshold;
+    private int size;
+    private Node<K,V>[] table;
+
+    public MyHashMap() {
+        createNewTable();
```

Comment:
for example you can change this part of the code to:
```java
hashTable = (Node<K, V>[]) new Node[DEFAULT_CAPACITY];
threshold = (int)(DEFAULT_CAPACITY * DEFAULT_LOAD_FACTOR);
``` 
consideration near the implementation of the createNewTable method